game/world/triggers/cscamera.cpp

```diff
@@ -44,8 +45,6 @@ CsCamera::KbSpline::KbSpline(const std::vector<std::shared_ptr<zenkit::VCameraTr
     kF.c[0] =  2 * p1 - 2*p2 +   dd + sd;
     kF.c[1] = -3 * p1 + 3*p2 - 2*dd - sd;
     kF.c[2] = std::move(dd);
-
-    pathLength += kF.arcLength();
```

Comment:
arclength function  is unused now.

Reply:
Intentional - let's keep arcLength for now.
Also this implementation is very trivial - it just tessellates curve into 100 linear segments and sum their length.